bin/python_scripts/remove_invalid_tags.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Set up environment from ckan.ini
+# export POSTGRES_URL=<sqlalchemy.url from ckan.ini>
+#
+# Execute script like this -
+# python remove_invalid_tags.py
+#
+
+import csv, io, logging, os, psycopg2, subprocess, sys
+from datetime import datetime
+
+from ckan.logic.schema import default_tags_schema
+from ckan.lib.navl.dictization_functions import validate
+from ckan.model.types import make_uuid
+
+POSTGRES_URL = os.environ['POSTGRES_URL']
+
+logger = logging.getLogger(__name__)
+connection = psycopg2.connect(POSTGRES_URL)
+
+
+def setup_logging():
+    _format = logging.Formatter('%(asctime)s - %(levelname)s - %(message)s')
+    logger.setLevel(logging.INFO)
+    c_handler = logging.StreamHandler()
+    c_handler.setFormatter(_format)
+    logger.addHandler(c_handler)
+
+    f_handler = logging.FileHandler('remove_invalid_tags.log')
+    f_handler.setFormatter(_format)
+    logger.addHandler(f_handler)
+
+    logger.info('====================================================================')
+
+
+def is_dev():
+    return '@db' in POSTGRES_URL
+
+
+def remove_invalid_tags(live=False):
```

Comment:
It might be good to split this function into smaller functions, but I realise that this script will probably only be run once so perhaps it's not too important.

Reply:
I think I'll leave it for now